example/histogram.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2023 Jiri Nytra (jiri.nytra at gmail dot com)
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <flux.hpp>
+
+#include <cstddef>
+#include <ostream>
+#include <string>
+#include <random>
+#include <iostream>
+#include <iomanip>
+#include <map>
+
+template<typename T>
+flux::generator<int> randu(T min, T max)
+{
+    std::mt19937 rng;
+    std::uniform_int_distribution dist(min, max);
+
+    while (true)
+        co_yield dist(rng);
```

Comment:
Braces again :) (I missed this last time, sorry)

```suggestion
    while (true) {
        co_yield dist(rng);
    }
```